src/systems/systems/src/hydraulic/mod.rs

```diff
@@ -141,13 +152,7 @@ impl PowerTransferUnit {
         } else if delta_p.get::<psi>() > 500. || (self.is_active_left && delta_p.get::<psi>() > 5.)
         {
             // Left sends flow to right
-            let mut vr = 16.0f64.min(loop_left.loop_pressure.get::<psi>() * 0.0058) / 60.0;
-
-            // Limiting available flow with maximum flow capacity of all pumps of the loop.
-            // This is a workaround to limit PTU greed for flow
-            vr = vr.min(
-                loop_left.current_max_flow.get::<gallon_per_second>() * Self::AGRESSIVENESS_FACTOR,
-            );
+            let mut vr = 16.0f64.min(loop_left_pressure.get::<psi>() * 0.0058) / 60.0;
```

Comment:
Maybe we can use a constant for these magic numbers. The unit conversion factors are fine, but I don't really know what `16.0` and `34.0` (below) represent. Also perhaps we should use something more verbose instead of `vr`.

Reply:
Renamed and added a comment.
Hope to remove all of this in future PR simulating the whole PTU physics